packages/aws-cdk-lib/aws-cognito/lib/user-pool.ts

```diff
@@ -805,6 +811,7 @@ export interface IUserPool extends IResource {
 abstract class UserPoolBase extends Resource implements IUserPool {
   public abstract readonly userPoolId: string;
   public abstract readonly userPoolArn: string;
+  public abstract readonly userPoolProviderName: string;
```

Comment:
Same comment as left in `IUserPool.userPoolProviderName`. Does this field need to be mandatory for the new logic to work?

Reply:
Same update as above: no issue here